assets/textures/gui/bevel/Energy.svg.import

```diff
@@ -0,0 +1,37 @@
+[remap]
+
+importer="texture"
+type="CompressedTexture2D"
+uid="uid://cslhvonq2rf1y"
+path="res://.godot/imported/Energy.svg-1f7d18df827b7946dc1bd6a678230948.ctex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://assets/textures/gui/bevel/Energy.svg"
+dest_files=["res://.godot/imported/Energy.svg-1f7d18df827b7946dc1bd6a678230948.ctex"]
+
+[params]
+
+compress/mode=0
+compress/high_quality=false
+compress/lossy_quality=0.7
+compress/hdr_compression=1
+compress/normal_map=0
+compress/channel_pack=0
+mipmaps/generate=true
+mipmaps/limit=-1
+roughness/mode=0
+roughness/src_normal=""
+process/fix_alpha_border=true
+process/premult_alpha=false
+process/normal_map_invert_y=false
+process/hdr_as_srgb=false
+process/hdr_clamp_exposure=false
+process/size_limit=0
+detect_3d/compress_to=1
+svg/scale=1.0
```

Comment:
SVG scale should be set to 0.25, and import size limit to 256 (with the original svg defining its size as 512x512) so that there are no weird border artifacts when the icon is displayed in the game.

Reply:
Right, I just left the import at the default values. I'll be mindful of that when contributing icons in the future.
I updated it, should be good now.